VocaDb.ResXFileCodeGenerator.Tests/GithubIssues/Issue3/GeneratorTests.cs

```diff
@@ -70,8 +70,8 @@ public void Generate_StringBuilder_Name_NotValidIdentifier()
   <resheader name=""writer"">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name=""Invalid identifier {0}"" xml:space=""preserve"">
-    <value>String '{0}' is not a valid identifier.</value>
+  <data name=""1InvalidId"" xml:space=""preserve"">
+    <value>Identifier cannot start with a number.</value>
```

Comment:
Is there any particular reason why you changed these unit tests rather than adding new ones?